### PR TITLE
fixed fonts preloading

### PR DESCRIPTION
**Pull request by IntegralTeam**
Status: [WIP][WAITING FOR REVIEW]

Description:
Finally solves font preloading with proper font loader rules

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -52,6 +54,7 @@
 
     <!-- Global site tag (gtag.js) - Google Analytics -->
     <script async src="https://www.googletagmanager.com/gtag/js?id=UA-104311073-2"></script>
+
```
Comment (BeOleg) [inline public/index.html]: ?

File: public/index.html [hunk 2]
```diff
@@ -26,17 +26,19 @@
...
+    <link rel="preload" href="/static/media/material-icons.woff2" as="font" crossorigin="anonymous" />
+    <link rel="preload" href="/static/media/cryptocoins.woff2" as="font" crossorigin="anonymous" />
 
     <script type="text/javascript">window._trackJs = { token: 'd378b378ded441aaa0f23b5403f62a53' };</script>
 
     <!-- <script type="text/javascript" src="https://cdn.trackjs.com/releases/current/tracker.js"></script> -->
 
-    <link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/cookieconsent2/3.0.3/cookieconsent.min.css" />
-    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.0.13/css/all.css" integrity="sha384-DNOHZ68U8hZfKXOrtjWvjxusGo9WQnrNx2sqG0tfsghAvtVlRW3tvkXWZh58N9jp" crossorigin="anonymous">
+    <link as="style" rel="preload" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/cookieconsent2/3.0.3/cookieconsent.min.css" />
```
Comment (BeOleg) [inline public/index.html]: Seems like we have cookieconsent.min.js twice

File: public/index.html [hunk 3]
```diff
@@ -80,6 +83,6 @@
     </noscript>
     <div id="root"></div>
 
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/cookieconsent2/3.0.3/cookieconsent.min.js"></script>
+    <script async src="https://cdnjs.cloudflare.com/ajax/libs/cookieconsent2/3.0.3/cookieconsent.min.js"></script>
```
Comment (BeOleg) [inline public/index.html]: Seems like we load this script twice
https://github.com/onitsoft/nexchange-open-client-react/pull/694/files#r388313703

Comment (BeOleg) [reply public/index.html]: @IntegralTeam

File: src/components/FAQ/FAQ.scss [hunk 4]
```diff
@@ -51,7 +51,7 @@
     margin-bottom: 0px;
     font-weight: 400;
     font-size: 16px;
-    font-family: 'Clan Offc Pro Book';
+    font-family: 'Clan Offc Pro Book', -apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,"Helvetica Neue",Arial,"Noto Sans",sans-serif,"Apple Color Emoji","Segoe UI Emoji","Segoe UI Symbol","Noto Color Emoji";
```
Comment (BeOleg) [inline src/components/FAQ/FAQ.scss]: Can you please explain this diff @IntegralTeam specifically what `-apple-system` does here

File: src/components/FAQ2/FAQ.scss [hunk 5]
```diff
@@ -51,7 +51,8 @@
     margin-bottom: 0px;
     font-weight: 400;
     font-size: 16px;
-    font-family: 'Clan Offc Pro Book';
+    font-family: 'Clan Offc Pro Book', -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, 'Helvetica Neue', Arial, 'Noto Sans',
```
Comment (BeOleg) [inline src/components/FAQ2/FAQ.scss]: Same as https://github.com/onitsoft/nexchange-open-client-react/pull/694/files#r388314935

File: src/components/FAQ2/FAQ.scss [hunk 6]
```diff
@@ -88,41 +88,44 @@
...
         font-size: 26px !important;
         margin-bottom: 26px !important;
         padding-left: 32px !important;
       }
-    
-      input::-webkit-input-placeholder { /* WebKit browsers */
+
+      input::-webkit-input-placeholder {
+        /* WebKit browsers */
         font-size: 26px !important;
```
Comment (BeOleg) [inline src/components/FAQ2/FAQ.scss]: @IntegralTeam 
I know it is not your diff but if possible, please get rid of the 'important at some point'

File: src/components/FAQ2/FAQ.scss [hunk 7]
```diff
@@ -88,41 +88,44 @@
...
       }
-    
-      input:-moz-placeholder { /* Mozilla Firefox 4 to 18 */
+
+      input:-moz-placeholder {
+        /* Mozilla Firefox 4 to 18 */
         font-size: 26px !important;
```
Comment (BeOleg) [inline src/components/FAQ2/FAQ.scss]: same as https://github.com/onitsoft/nexchange-open-client-react/pull/694/files#r388315737

File: src/components/FAQ2/FAQ.scss [hunk 8]
```diff
@@ -88,41 +88,44 @@
...
       }
-    
-      input::-moz-placeholder { /* Mozilla Firefox 19+ */
+
+      input::-moz-placeholder {
+        /* Mozilla Firefox 19+ */
         font-size: 26px !important;
```
Comment (BeOleg) [inline src/components/FAQ2/FAQ.scss]: same as https://github.com/onitsoft/nexchange-open-client-react/pull/694/files#r388315737

File: src/components/FAQ2/FAQ.scss [hunk 9]
```diff
@@ -88,41 +88,44 @@
...
       }
-    
-      input:-ms-input-placeholder { /* Internet Explorer 10+ */
+
+      input:-ms-input-placeholder {
+        /* Internet Explorer 10+ */
         font-size: 26px !important;
```
Comment (BeOleg) [inline src/components/FAQ2/FAQ.scss]: same as https://github.com/onitsoft/nexchange-open-client-react/pull/694/files#r388315737

File: public/index.html (hunk 2 above)
Comment (IntegralTeam) [reply public/index.html]: First one is JS, another one is CSS.

File: public/index.html (hunk 3 above)
Comment (IntegralTeam) [reply public/index.html]: First one is JS, another one is CSS.

File: src/components/FAQ/FAQ.scss (hunk 4 above)
Comment (IntegralTeam) [reply src/components/FAQ/FAQ.scss]: Every font, loading we it or not, should has fallback font-faces. That's the rule of good markup.

File: src/components/FAQ2/FAQ.scss (hunk 5 above)
Comment (IntegralTeam) [reply src/components/FAQ2/FAQ.scss]: Same reply

File: public/index.html (hunk 1 above)
Comment (IntegralTeam) [reply public/index.html]: Is that a problem?

File: src/components/FAQ2/FAQ.scss (hunk 6 above)
Comment (IntegralTeam) [reply src/components/FAQ2/FAQ.scss]: !important, itself, isn't bad. And when you're fixing user-agent styles it's not bad too.

**Pull request closed without merging**